docs/introduction.md

```diff
@@ -0,0 +1,125 @@
+---
+id: introduction
+title: Introduction
+---
+
+Many different kinds of people use React Native: from advanced iOS developers to React JS beginners, to people getting started programming for the first time in their career. These docs were written for all learners, no matter their experience level or background.
+
+## How to use these docs
+
+You can start here and read through these docs linearly like a book; or you can read the specific sections you need. Already familiar with React JS? You can skip that section—or read it for a light refresher.
+
+## Prerequisites
+
+To work with React Native, you will need to have an understanding of JavaScript fundamentals. If you’re new to JavaScript or need a refresher, you can [dive in](https://developer.mozilla.org/en-US/docs/Web/JavaScript) or [brush up](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) at Mozilla Developer Network.
+
+> While we do our best to assume no prior knowledge of React, iOS, or Android development, these are valuable topics of study for the aspiring React Native developer. Where sensible, we have linked to resources and articles that go more in depth.
+
+### Environment Setup
```

Comment:
I recommend moving this after "Interactive examples". Reading from the top down it wasn't clear to me what the next step was: should I click this link and go somewhere else or continue to scroll down? I think it's best to get people to place with RN asap, and then recommend "oh, if you don't wanna play with it here on the website, set up a local dev env here […] and then continue the guide."

Reply:
So interestingly enough, it _was_ further down, but someone suggested it should be closer to the top for readers who want to set up their environment before before getting too far along. I tend to agree with you, though. I think the straighter flow is better. Let me see what Dan's feedback says :)

Reply:
I don’t care strongly either way. Trust you to choose this!

Reply:
I'm going to leave the dev environment stuff in a callout under the interactive examples. One reference should be enough.